Arena/arena.py

```diff
@@ -0,0 +1,41 @@
+import pygame
+
+from Arena.terrain import Grass, Mud, Stone, Water
+
+
+class Arena():
+    def __init__(self) -> None:
+        self.arenaSize = pygame.Vector2(1000, 1000)
+        self.tileSize = pygame.Vector2(50, 50)
```

Comment:
The `arenaSize` should either be passed via game state or removed from the game state.

The `tileSize` should definitely be removed from here and either be kept in the `Terrain` class only, or in the game state only.